cli/cmd/install.go

```diff
@@ -839,7 +837,7 @@ func render(w io.Writer, values *l5dcharts.Values) error {
 			Name:      addOn.Name(),
 			Dir:       filepath.Join(addOnChartsPath, addOn.Name()),
 			Namespace: controlPlaneNamespace,
-			RawValues: append(rawValues, addOn.Values()...),
+			RawValues: append(addOn.Values(), rawValues...),
```

Comment:
This change was done as we were running into weird cases because of https://stackoverflow.com/questions/35920534/why-does-append-modify-passed-slice. As rawValues containing `addon.` fields directly causing conflicts with other add-ons with similar fields.

As `append.Values()` returns a copy of the slice, this should not update the underlying addOn.Values(), and cause any weird cases!